bigchaindb/crypto/fulfillments/ed25519_sha256.py

```diff
@@ -0,0 +1,266 @@
+import base58
+
+from bigchaindb.crypto.ed25519 import Ed25519VerifyingKey
+from bigchaindb.crypto.fulfillments.base_sha256 import BaseSha256Fulfillment
+from bigchaindb.crypto.buffer import Predictor
+
+
+class Ed25519Sha256Fulfillment(BaseSha256Fulfillment):
```

Comment:
the attributes may be assigned after construction, hence the checks on the setter.
The getters (and non-checked props) are merely to be consistent